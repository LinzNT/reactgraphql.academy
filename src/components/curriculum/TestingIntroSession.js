import React from 'react'
import Ul, { Li } from '../layout/Ul'
import Session from './Session'

const TestingIntroSession = ({ hideTitle }) => (
  <Session hideTitle={hideTitle} title="Testing Principales">
    <Ul>
      <Li>Build your own test framework in JavaScript</Li>
      <Li>Testing Redux using Jest</Li>
      <Li>Mocking and testing API calls</Li>
      <Li>White-box testing Vs. Black-box testing</Li>
    </Ul>
  </Session>
)

export default TestingIntroSession
