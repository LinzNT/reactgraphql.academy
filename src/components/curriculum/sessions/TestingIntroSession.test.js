import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import TestingIntroSession from './TestingIntroSession'

describe('<TestingIntroSession />', () => {
  it('should render component', () => {
    const wrapper = shallow(<TestingIntroSession />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
