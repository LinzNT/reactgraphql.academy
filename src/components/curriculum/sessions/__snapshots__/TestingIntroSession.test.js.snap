// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TestingIntroSession /> should render component 1`] = `
<Session>
  <styled.ul>
    <styled.li>
      Build your own test framework in JavaScript
    </styled.li>
    <styled.li>
      Testing Redux using Jest
    </styled.li>
    <styled.li>
      Mocking and testing API calls
    </styled.li>
    <styled.li>
      White-box testing Vs. Black-box testing
    </styled.li>
  </styled.ul>
</Session>
`;
